protocol-units/bridge/integration-tests/tests/eth_movement.rs

```diff
@@ -0,0 +1,155 @@
+	let anvil = Anvil::new().port(harness.rpc_port()).spawn();
+
+	// Set a funded signer
+	let _ = harness.set_eth_signer(anvil.keys()[0].clone());
+
+	let initiator_address = harness.deploy_initiator_contract().await;
+	let expected_address = address!("5fbdb2315678afecb367f032d93f642f64180aa3");
+
+	assert_eq!(initiator_address, expected_address);
+}
+
+#[tokio::test]
+async fn test_client_should_successfully_call_initialize() {
+	let mut harness: TestHarness = TestHarness::new_only_eth().await;
+	let anvil = Anvil::new().port(harness.rpc_port()).spawn();
+
+	let _ = harness.set_eth_signer(anvil.keys()[0].clone());
+	harness.deploy_init_contracts().await;
+}
+
+#[tokio::test]
+async fn test_client_should_successfully_call_initiate_transfer() {
+	let mut harness: TestHarness = TestHarness::new_only_eth().await;
+	let anvil = Anvil::new().port(harness.rpc_port()).spawn();
+
+	//Set a funded signer
+	let signer_address = harness.set_eth_signer(anvil.keys()[0].clone());
+
+	harness.deploy_init_contracts().await;
+
+	let recipient = harness.gen_aptos_account();
+	let hash_lock: [u8; 32] = keccak256("secret".to_string().as_bytes()).into();
+
+	harness
+		.eth_client_mut()
+		.expect("Failed to get EthClient")
+		.initiate_bridge_transfer(
+			InitiatorAddress(EthAddress(signer_address)),
+			RecipientAddress(recipient),
+			HashLock(hash_lock),
+			TimeLock(100),
+			Amount(1000), // Eth
+		)
+		.await
+		.expect("Failed to initiate bridge transfer");
+}
+
+#[tokio::test]
+#[ignore] // To be tested after this is merged in https://github.com/movementlabsxyz/movement/pull/209
+async fn test_client_should_successfully_get_bridge_transfer_id() {
+	let mut harness: TestHarness = TestHarness::new_only_eth().await;
+	let anvil = Anvil::new().port(harness.rpc_port()).spawn();
+
+	// Set a funded signer
+	let signer_address = harness.set_eth_signer(anvil.keys()[0].clone());
+	harness.deploy_init_contracts().await;
+
+	let recipient = harness.gen_aptos_account();
+	let hash_lock: [u8; 32] = keccak256("secret".to_string().as_bytes()).into();
+
+	harness
+		.eth_client_mut()
+		.expect("Failed to get EthClient")
+		.initiate_bridge_transfer(
+			InitiatorAddress(EthAddress(signer_address)),
+			RecipientAddress(recipient),
+			HashLock(hash_lock),
+			TimeLock(100),
+			Amount(1000), // Eth
+		)
+		.await
+		.expect("Failed to initiate bridge transfer");
+
+	//TODO: Here call get details with the captured event
+}
+
+#[tokio::test]
+#[ignore] // To be tested after this is merged in https://github.com/movementlabsxyz/movement/pull/209
+async fn test_client_should_successfully_complete_transfer() {
+	let mut harness: TestHarness = TestHarness::new_only_eth().await;
+	let anvil = Anvil::new().port(harness.rpc_port()).spawn();
+
+	// set funded signer
+	let signer_address = harness.set_eth_signer(anvil.keys()[0].clone());
+	harness.deploy_init_contracts().await;
+
+	//some data to set for the recipient.
```

Comment:
This comment doesn't seem necessary as the next couple lines are self-describing and the comment doesn't add much in the way of clarification.

Reply:
Will remove